### DISCUSSION TITLE
Filter pages by author

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#302] [Anything Else]
Multi-author site that wants to be able to filter Top Pages report by author in order to see all the pages published by a specific author, combined number of views for those pages, number of views on individual posts published by that author and so on.

(request via email)

**Comment by metmarkosaric** [comment]
We're now opened up a beta version of this to everyone. You can attach custom properties (also known as custom dimensions in Google Analytics) when sending a pageview to Plausible in order to create custom metrics that allow you to track pages by author and similar stats. Instructions at https://plausible.io/docs/custom-pageview-props

**Reply by sn3h** [reply]
Well, what if soemthing is coauthored? Whould we report it as `"authors" : "<author1>,<author2>"`, or `<author1>:"true"` ... ? This is good tool but missing arrays or some other option to pass multiple values so much...
